example_dags/example_amazon_s3_snowflake_transform.py

```diff
@@ -46,19 +46,33 @@ def aggregate_data(df: pd.DataFrame):
     catchup=False,
 )
 def example_amazon_s3_snowflake_transform():
+    input_table_1 = Table(
+        "ADOPTION_CENTER_1",
+        database=os.environ["SNOWFLAKE_DATABASE"],
+        schema=os.environ["SNOWFLAKE_SCHEMA"],
+        conn_id="snowflake_conn",
+    )
+    input_table_2 = Table(
+        "ADOPTION_CENTER_2",
+        database=os.environ["SNOWFLAKE_DATABASE"],
+        schema=os.environ["SNOWFLAKE_SCHEMA"],
+        conn_id="snowflake_conn",
+    )
+
+    temp_table_1 = aql.load_file(
+        path="s3://tmp9/ADOPTION_CENTER_1.csv",
+        file_conn_id="",
+        output_table=input_table_1,
+    )
+    temp_table_2 = aql.load_file(
+        path="s3://tmp9/ADOPTION_CENTER_2.csv",
+        file_conn_id="",
+        output_table=input_table_2,
+    )
+
```

Comment:
@utkarsharma2 would it be an option to use that table fixture, so we don't leave trash behind, after running the test?

Reply:
@tatiana Yes, It's an issue that we leave the temp tables behind. But I think we need a different way to clear them since they are example dags that are meant to be used by end-Users and adding any kind of fixtures means they also need the same setup to run them.

Reply:
Fair point, @utkarsharma2 !